docs/source/tutorials/improving_ml_performance.ipynb

```diff
@@ -0,0 +1,3573 @@
+       "4  d57e1a    92.0    79.0    98.0  great final presentation +10   \n",
+       "\n",
+       "  noisy_letter_grade  \n",
+       "0                  B  \n",
+       "1                  F  \n",
+       "2                  F  \n",
+       "3                  B  \n",
+       "4                  A  "
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df_train = pd.read_csv(\n",
+    "    \"https://cleanlab-public.s3.amazonaws.com/Datasets/student-grades/clos_train_data.csv\"\n",
+    ")\n",
+    "\n",
+    "df_test = pd.read_csv(\n",
+    "    \"https://cleanlab-public.s3.amazonaws.com/Datasets/student-grades/clos_test_data.csv\"\n",
+    ")\n",
+    "\n",
+    "df_train.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aa2ca8e2-71a9-4699-80b3-352c325ae8e3",
+   "metadata": {},
+   "source": [
+    "## 2. Preprocess the dataset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b39cd525-7a09-4d8e-811e-44ac1072f438",
+   "metadata": {},
+   "source": [
+    "Before training a ML model, we preprocess our dataset. The type of preprocessing that is best will depend on what ML model you use. This tutorial will demonstrate an XGBoost model, so we'll process the **notes** and **noisy_letter_grade** columns into categorical columns for this model (each category encoded as an integer). You can alternatively use [Cleanlab Studio](https://cleanlab.ai/blog/data-centric-ai/), which will automatically produce a high-accuracy ML model for your raw data, without you having to worry about any ML modeling or data preprocessing work."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "1b5f50e6-d125-4e61-b63e-4004f0c9099a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create label encoders for the categorical columns\n",
+    "grade_le = preprocessing.LabelEncoder()\n",
+    "notes_le = preprocessing.LabelEncoder()\n",
+    "\n",
+    "# Process the feature columns\n",
+    "train_features = df_train.drop([\"stud_ID\", \"noisy_letter_grade\"], axis=1).copy()\n",
+    "train_features[\"notes\"] = notes_le.fit_transform(train_features[\"notes\"])\n",
+    "train_features[\"notes\"] = train_features[\"notes\"].astype(\"category\")\n",
+    "\n",
+    "# Process the label column\n",
+    "train_labels = pd.DataFrame(grade_le.fit_transform(df_train[\"noisy_letter_grade\"].copy()), columns=[\"noisy_letter_grade\"])\n",
+    "\n",
+    "\n",
+    "# Keep separate copies of these training features and labels for later use\n",
+    "train_features_v2 = train_features.copy()\n",
+    "train_labels_v2 = train_labels.copy()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "750cd820-7565-4314-9dda-808cbe7c638f",
+   "metadata": {},
+   "source": [
+    "We first solely preprocessed the training data to avoid information leakage from our test set. Here's how the processed training features look:"
```

Comment:
```suggestion
    "We first solely preprocessed the training data to avoid information leakage (using test data information that would not be available at prediction time). Here's how the processed training features look:"
```

Reply:
what would be your suggestion?  I think [data preprocessing in ML](https://towardsdatascience.com/introduction-to-data-preprocessing-in-machine-learning-a9fa83a5dc9d) has a pretty standard meaning, but curious what term you would use

Reply:
also now realizing the module we're importing to do this is literally called `sklearn.preprocessing` so I'm unsure where your confusion about preprocessing is coming from